client/verta/verta/_internal_utils/_utils.py

```diff
@@ -27,6 +29,11 @@
 from ..external import six
 from ..external.six.moves.urllib.parse import urljoin  # pylint: disable=import-error, no-name-in-module
 
+from .credentials import (
+    EmailCredentials,
+    JWTCredentials,
+)
```

Comment:
With full awareness that our `import` ordering is kind of a mess everywhere, I'd put this a few lines below, next to `from . import importer`. But we could clean this all up in the future™.